### PR TITLE
remove external download material.min.js

### PR DESCRIPTION
**Pull request by binarytemple**
I've been working with drone in a docker-compose network. When I turn off my Wifi I found the drone-server was unusable, the reason is that the dropdown:

* Dashboard
* Account 
* Logout

Is dependent on material-ui, which is downloaded from a CDN. When offline it doesn't render anything so is unusable.

What I've done is to add 'material' as an NPM dependency (package.js) and configure webpack (webpack.config.js) to add it's contents to app.js. 

Let me know what you think.

### CODE REVIEW

**Comment by metalmatze** [comment]
I think you installed the wrong dependency.
You should run `npm install material-design-lite --save` for the right. This is taken from https://getmdl.io/started/
Otherwise LGTM.

**Comment by binarytemple** [comment]
Hi corrected the dependency as suggested, tested it locally using:

```
node webpack.devserver.js  --host=127.0.0.1:8000 --scheme=http --token=XXXXXXXXXXX
```

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 /*eslint-disable */
 import './index.less';
 
+const Material = require('material-design-lite')
```
Comment (tboerger) [inline src/index.js]: Mixing `import` and `require`? :-1:

File: webpack.config.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ var webpack = require('webpack');
 module.exports = {
   entry: {
     app: [
+      'react-mdl/extra/material.min.js',
```
Comment (tboerger) [inline webpack.config.js]: Why? Just integrate it into the bundle.

**Pull request closed without merging**